### ISSUE TITLE
Most of the files are identical?

### ISSUE THREAD

**Issue by thomask77** [psxdev/luke_firmwares#1]
Hi!

I noticed that the .bin files actually contain just 5 different firmware versions:

```bash
$ md5sum *.bin | sort   
24afe3c941dc599aafd23ac5170a4d14  600_B0A82F_68032.bin
24afe3c941dc599aafd23ac5170a4d14  650_B0CFC0_68032.bin
3a6ff06e3ff57615c4019b0a5c2e47ca  101_84C4C0_66576.bin
3a6ff06e3ff57615c4019b0a5c2e47ca  105_7204C0_66576.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  101_85C8E0_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  105_7308E0_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  150b_748970_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  151_748970_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  152_748970_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  160_74C970_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  161_74C970_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  162_74C970_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  175_971DD0_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  176_858E30_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  315_9244D0_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  350_90F640_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  355_90FEE0_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  400_936B20_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  401_936A80_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  405_936D20_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  406_936E70_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  407_936FB0_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  455_A4FFD0_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  500_A71350_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  505_A71750_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  505_devkit_C53980_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  600_B1D8BE_64036.bin
99ca1e97ea2bb9edaf4c6f511292b4e2  650_B1F6E1_64036.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  350_8FEB30_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  355_8FF3D0_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  400_948380_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  401_9482E0_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  405_948580_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  406_9486D0_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  407_948810_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  455_A3E4E0_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  500_A82D00_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  505_A83100_67924.bin
9e7ddcb8b891d7227ac0bdc998ddbfe7  505_devkit_C65330_67924.bin
e70ad69b3faab612e8115960f568a924  150b_738530_66608.bin
e70ad69b3faab612e8115960f568a924  151_738530_66608.bin
e70ad69b3faab612e8115960f568a924  152_738530_66608.bin
e70ad69b3faab612e8115960f568a924  160_73C530_66608.bin
e70ad69b3faab612e8115960f568a924  161_73C530_66608.bin
e70ad69b3faab612e8115960f568a924  162_73C530_66608.bin
e70ad69b3faab612e8115960f568a924  175_961990_66608.bin
e70ad69b3faab612e8115960f568a924  176_8489F0_66608.bin
e70ad69b3faab612e8115960f568a924  315_913F00_66608.bin
```
Is this intentional?

**Comment by psxdev** [comment]
Yes indeed. These are the playstation camera firmwares inside each PlayStation kernel release that i have

**Comment by psxdev** [comment]
Check https://psxdev.github.io/luke.html

**Comment by thomask77** [comment]
Thank you!

Reason I'm asking: I'm trying to get a CUH-ZEY1 camera to work under Ubuntu 18.04.

When I'm using ps4eye_init.py from https://github.com/ps4eye/ps4eye/tree/master/python, the Camera gets initialized, but Linux applications just show a black window.

I've tried all the different Firmware images, but none work in Ubuntu :/

**Comment by psxdev** [comment]
Sorry It is not a firmware problem. Disable uvc kernel driver. Check dmesg.
Also there is better way instead of use that python code
https://github.com/psmoveservice/PSMoveService/issues/35#issuecomment-232315000

**Comment by thomask77** [comment]
I got it to work!! It was a combination of two problems:

1. I first tried to use the camera in Windows10 using VMware15 with a Ubuntu 18.04 guest. This doesn't work, but seems to be a VMware/USB3 problem.

2. Even when booting Ubuntu 18.04 from a Live USB-Stick, I initially had the empty-window problem. This comment solved the issue:
  https://github.com/bigboss-ps3dev/PS4EYECam/issues/1#issuecomment-284658830

Thanks again for your great work!
